dll/win32/iernonce/iernonce.cpp

```diff
@@ -40,16 +40,40 @@ RunOnceExProcess(_In_ HWND hwnd,
                  _In_ LPCSTR path,
                  _In_ int nShow)
 {
+    // ShellExecuteEx may use COM.
```

Comment:
Well.... that's actually a "known" fact. Just say you're using shell32 functionality.